cmd/registry-server/main.go

```diff
@@ -27,16 +27,30 @@ import (
 var config server.Config
 
 func main() {
-	configFlag := flag.String("c", "", "specify a configuration file")
+	configFlag := flag.String("c", "config/registry.yaml", "specify a configuration file")
 	flag.Parse()
-	if *configFlag != "" {
-		b, err := ioutil.ReadFile(*configFlag)
+	if path := *configFlag; path != "" {
```

Comment:
I didn't realize that following symbolic links was so difficult with the Go standard library! This seems like a lot to maintain just to be able to use links in the repo.

Reply:
I'm not very concerned about the maintenance, but I agree it's a bit difficult to handle. Unless this `os` library changes behavior underneath us this code shouldn't cause any problems, and if it does fail the errors will clearly point here.

As long as we have registry.yaml in the repository I think it's better to support it as a possible config for server startup. If we want to remove it we will have to change our containers are built.